app/config.js

```diff
@@ -56,8 +56,17 @@ module.exports = require('rc')('agregore', {
   hyperOptions: {
     storage: DEFAULT_HYPER_DIR
   },
-  // All options here: https://github.com/ssbc/ssb-config#configuration
-  ssbOptions: {},
+  /**
+   * All ssb options here: https://github.com/ssbc/ssb-config#configuration
+   * For bundled ssb server we use ssbd. Pass in an array of require'd ssb plugins
+   */
+  ssbOptions: {
+    appname: DEFAULT_SSB_APPNAME,
+    ssbd: {
+      runServer: true,
+      plugins: require('@metacentre/shipyard-ssb')
```

Comment:
If we can require absolute paths, we can probably require regular paths too, right?

```suggestion
      plugins: ['@metacentre/shipyard-ssb']
```

Reply:
This was ages ago but my brain says something-something-something-node-resolution-algorithm. I did this with shipyard and found it a nightmare. Honestly, it was much smoother just requiring a module that required the plugins you want.

Reply:
Cool, I'll use `require.resolve` instead then if that's okay

Reply:
Or honestly we can keep it for now. 🤷